ci/CMakeLists.txt

```diff
@@ -6,6 +6,14 @@
 # Alex Richert, 6 Dec 2023
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/CMakeModules/Modules)
+# Add ESMF's cmake files and library locations
+if(DEFINED ENV{ESMFMKFILE})
+  get_filename_component(TEMP_DIR $ENV{ESMFMKFILE} DIRECTORY)
+  get_filename_component(ESMF_ROOT ${TEMP_DIR} DIRECTORY)
+  list(APPEND CMAKE_MODULE_PATH ${ESMF_ROOT}/cmake)
+  include_directories(${ESMF_ROOT}/include)
+  link_directories(${TEMP_DIR})
```

Comment:
Why are these two (include_directories and link_directories) needed? If we use FindESMF.cmake from esmf's install tree, find_package (at line 65) should properly set esmf import targets.